src/en/about/guide.md

```diff
@@ -1,57 +1,95 @@
-# Lemmy Guide
+# Lemmy quick start guide
 
-Start typing...
+So you want to join Lemmy and get posting. Great! Here's how to dive straight in.
+
+## Where do I sign up?
+
+The first thing you have to do is **choose your server**. This is an extra step compared to sites like Reddit, but isn’t as difficult as it may seem.
+
+*Like with email, your identity is hosted by the server you sign up on. So for example, I joined lemmy.ml so to mention me you can type @retiolus@lemmy.ml in your post.*
+
+If what you want to talk about most fits into a clear category (maybe it’s video games or art or queer life or coding or fiction or whatever) then it might be worth making your first server one where that will primarily host that sort of content – it’ll be easier to make connections and find like-minded folk. Some consider your server to be like a neighbourhood or a venue, where the general chatter can have a specific focus.
+
+*You have the ability to view all public local posts that have been made by people on your server in the so-called “Local” tab.*
+
+If you aren’t here to stick mainly to one topic, you’ll likely want to pick a server that caters to general interests. Either way, there’s a helpful servers list on [join-lemmy.org](https://join-lemmy.org/instances).
+
+**Don’t panic!** You’ll be able to chatter to people on other servers, no matter which one you choose. Remember, this is like email, and you can still email your mum on her ancient Yahoo account from your GMail, for example.
```

Comment:
you'll **still** be able to interact with ...

The second sentence seems unnecessary as you already stated that above, but that's up to you.